test/controllers/api/v2/repositories_controller_test.rb

```diff
@@ -50,13 +50,15 @@ def assert_response_ids(response, expected)
     end
 
     def test_index_with_product_id
-      ids = Repository.where(:product_id => @product.id, :library_instance_id => nil).pluck(:id)
+      ids = Repository.where(:product_id => @product.id, :library_instance_id => nil).where(:content_view_version_id => @organization.default_content_view.versions.first.id).pluck(:id)
```

Comment:
@jlsherrill I couldn't see a way around returning these ids without a content view version query, it always returns one more repo fixture than is necessary (fedora_17_library_non_default iirc) if I don't add that at the end. I think its because our fixtures aren't a perfect real world replication of an environment (and are a bit of a house of cards, change one thing, everything breaks ;) Point being, do you think this is ok?